R/data_fetching.R

```diff
@@ -298,3 +298,60 @@ read_sql_db <- function(dbname, host, user, password) {
     }
   )
 }
+
+get_housing_data <- function() {
+
+  filename <-
+    system(
+      'bash get_latest_file.sh data_in/redcap/testing/STUDENT_HOUSING_DATA_TO_REDCAP_',
```

Comment:
```suggestion
      'bash get_latest_file.sh data_in/redcap/STUDENT_HOUSING_DATA_TO_REDCAP_',
```
Switching to the production data feed.